package.json

```diff
@@ -35,8 +35,13 @@
     "lru-cache": "^4.0.1",
     "morgan": "^1.7.0",
     "react": "^15.3.1",
+    "react-bootstrap": "^0.30.3",
```

Comment:
We don't need this as we already import bootstrap's JS and CSS manually [here](https://github.com/SivanMehta/wander/blob/master/public/index.html#L10)